### DISCUSSION TITLE
Localization, change ui language

### DISCUSSION THREAD

**Discussion by AndiLeni** [maplibre/maplibre-gl-js#4350] [Q&A] [answered]
Hello,

I was not able to find any information for this topic.
I need to change maplibres ui language from english to german, basically translating those strings: https://github.com/maplibre/maplibre-gl-js/blob/a4ce3ab94535f31d70bf7929fe87bc382683c89d/src/ui/default_locale.ts

Is this somehow possible? 
I am using the cdn version currently if this makes a difference.

Thanks and kind regards

**Comment by HarelM** [comment, marked as answer]
The following example has a locale settings: 
https://maplibre.org/maplibre-gl-js/docs/examples/cooperative-gestures/

**Reply by HarelM** [reply]
You can also update the map _locale variable content, but it's more of a hack, there isn't a proper public API to address this 
Feel free to open a PR to add this....

**Reply by AndiLeni** [reply]
Thanks. I used the `locale` property in the Map constructor. This is perfectly fine for my usecase :)
